src/pages/rasa.tsx

```diff
@@ -37,12 +46,36 @@ const Body = styled.div`
   overflow: hidden;
 `
 
-export const RasaPage: React.FC = () => {
-  const { t, i18n } = useTranslation()
+interface Metadata {
+  [key: string]: string
+}
+
+// TODO: change intent from env to remove / and pass it right here
+const createPayload = (intent: string, metadata: Metadata) => {
+  return `/${intent}{"${RASA_METADATA_ENTITY}":${JSON.stringify(metadata)}}`
```

Comment:
At this point I'm not sure what we gain from having everything in env variables, I mean what we would need to this is a true template string but I don't think this is an appropriate substitute.

Reply:
All right - will remove intent/entity/parameter env variables. Those should not really be changing anyway.